mobile.css

```diff
@@ -2836,9 +2846,9 @@ button:active {
     align-items: center;
     padding: 0;
     position: absolute;
-    width: 90%;
+    width: 100%;
     height: 274px;
-    left: 2%;
+    left: 0;
     top: 343px;
   }
 
```

Comment:
Your hover buttons is missing contrast.
Kindly work on it:
![Screen Shot 2021-09-28 at 9 49 10](https://user-images.githubusercontent.com/22258274/135111349-beb82c18-5e7d-4837-a952-b61828b946c8.png)

Reply:
Can you also offer any advice on what colours to use for my hover buttons as I have used the colours requested on the template?